packages/salesforcedx-vscode-core/test/vscode-integration/commands/templates/baseTemplateCommand.test.ts

```diff
@@ -0,0 +1,282 @@
+import { expect } from 'chai';
+import * as path from 'path';
+import * as sinon from 'sinon';
+import * as vscode from 'vscode';
+import { nls } from '../../../../src/messages';
+
+import {
+  CancelResponse,
+  ContinueResponse,
+  DirFileNameSelection
+} from '@salesforce/salesforcedx-utils-vscode/out/src/types';
+import {
+  BundlePathStrategy,
+  DefaultPathStrategy,
+  FilePathExistsChecker
+} from '../../../../src/commands/templates/baseTemplateCommand';
+import {
+  APEX_CLASS_EXTENSION,
+  AURA_DEFINITION_FILE_EXTS
+} from '../../../../src/commands/templates/metadataTypeConstants';
+import { notificationService } from '../../../../src/notifications';
+
+describe('SourcePathStrategy', () => {
+  it('successfully creates the bundle path', () => {
+    const bundlePathStrategy = new BundlePathStrategy();
+    const expectedPath = path.join('test-dir', 'TestCmp', 'TestCmp.cmp');
+    expect(
+      bundlePathStrategy.getPathToSource('test-dir', 'TestCmp', '.cmp')
+    ).to.equal(expectedPath);
+  });
+
+  it('successfully creates a default source path', () => {
+    const defaultPathStrategy = new DefaultPathStrategy();
+    const expectedPath = path.join('test-dir', 'TestClass.cls');
+    expect(
+      defaultPathStrategy.getPathToSource('test-dir', 'TestClass', '.cls')
+    ).to.equal(expectedPath);
+  });
+});
+
+describe('FilePathExistsChecker for Aura bundle', () => {
```

Comment:
The tests from this line to the end of the file were moved from `commands.test.ts`. With more time, I would have assessed whether we should be keeping these tests and whether or not more tests need to be added.